src/Settings.hs

```diff
@@ -114,3 +114,8 @@ programPath context@Context {..} = do
         install <- (\l -> l == stage || package == ghc) <$> maybeLatest
         let path = if install then installPath package else buildPath context
         return $ path -/- programName context <.> exe
+
+destDir :: Action FilePath
```

Comment:
Answering the second question: perhaps this will be spotted, because we track command lines. (Or maybe not?)